webshare/idlharness.https.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>WebShare IDL test</title>
+    <link rel="help" href="https://wicg.github.io/web-share/">
+    <script src=/resources/testharness.js></script>
+    <script src=/resources/testharnessreport.js></script>
+    <script src=/resources/WebIDLParser.js></script>
+    <script src=/resources/idlharness.js></script>
+  </head>
+  <body>
+    <script type="text/plain" id="untested">
+        interface Navigator {
+        };
+    </script>
+    <script type="text/plain" id="tested">
+        partial interface Navigator {
```

Comment:
Can you put this in interfaces/web-share.idl instead? https://github.com/w3c/web-platform-tests/commit/883229582ad98ed9af56c77b9a5ce23dbd00bbcf is an example of it, and I hope it'll make it easier to eventually automatically update these IDL files from specs.